pkg/format/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apk add --no-cache --initdb -p /out \
     musl \
     sfdisk \
     util-linux \
+    blkid \
```

Comment:
the alpine package `blkid` is required
- the `busybox` version returns exitcode 0 when content is found
- the `blkid` package version returns exitcode 2 when content is found on the block device (distinct result)